src/components/Shared/Filters/FilterPanel.tsx

```diff
@@ -155,6 +154,14 @@ export const FilterPanel: React.FC<FilterPanelProps & BoxProps> = ({
     FinancialAccountContext,
   ) as FinancialAccountType;
 
+  const matchFilterContactStatuses = (status: string | null | undefined) => {
```

Comment:
Could this use `convertContactStatus`?

Reply:
`ContactFilterStatusEnum` is different than `StatusEnum` because it includes `NULL`, `HIDDEN`, and `ACTIVE`. So this check is for those statuses that are missing from `StatusEnum`

Reply:
That makes sense. Can you remove the `String` coercion here too unless there's something I'm missing and that's necessary?